heartbeat/monitors/active/dialchain/builder.go

```diff
@@ -183,26 +184,27 @@ func makeEndpointJobs(
 
 	// Create job that first resolves one or multiple IP (depending on
 	// config.Mode) in order to create one continuation Task per IP.
-	jobName := jobName(typ, scheme, endpoint.Host, endpoint.Ports)
-	settings := monitors.MakeHostJobSettings(jobName, endpoint.Host, mode).WithFields(fields)
+	jobID := jobID(typ, scheme, endpoint.Host, endpoint.Ports)
+	settings := monitors.MakeHostJobSettings(jobID, endpoint.Host, mode)
```

Comment:
That's because those fields used to be part of the job settings. A general theme in this patch is applying Job level stuff higher up in the call stack, whereas before those options were passed in to a deep point, then used from `annotate`. The wrapping strategy now used is more clear IMHO.